admin/workflow_wiki/README.md

```diff
@@ -412,9 +422,276 @@ and likely will be too dense to remember everything when you first read it.
       or SAMs.
 
 ## Merging Code Changes into Main
+This section describes the team's workflow starting right after you have finished making your code change to merging that code into `upstream/main`.
 
 ### Assumptions
+We make the following assumptions in this tutorial.
+If any of them are not true, please resolve this before proceeding.
+1. All [assumptions from Making Code Changes](#assumptions) apply to this section as well.
+2. You have completed a code change via the guidelines laid out [previously](#making-code-changes).
 
 ### Details
+In order to merge a code change into `upstream`, you need to create a pull request.
+After creating a pull request, you need to pass the automated checks and get 3 review approvals.
+In order to get 3 approvals, you will need to address others' review comments and, potentially, make edits to your code.
+Once you have done this, you will be able to squash your code into a single final commit and merge in your code.
+
+1. Create a pull request from `<your github username>/<your branch name>` to `umrover/main`.
+    * Use the GitHub web page to create this pull request starting from your forked repository (`https://github.com/<your github username>/mrover-workspace`).
+    * Navigate to the `pull requests` tab on your fork (`https://github.com/<your github username>/mrover-workspace/pulls`).
+    * Then click `new pull request` (`https://github.com/<your github username>/mrover-workspace/compare`).
+    * Verify/set the base and head repositories and branches.
```

Comment:
"base should be umrover/main or something similar, while head should be your fork's branch..." ?
Just to make sure this doesn't get reversed accidentally?

Reply:
Yes. This is the terminology used on the GitHub page so I think it will be less confusing when you actually do it. You can also look at the images in the example below. Let me know if it's still not clear with what's written.